core/vm/actor/storage_power_actor.hpp

```diff
@@ -20,6 +27,165 @@ namespace fc::vm::actor {
     CHECK_PROOF_SUBMISSIONS
   };
 
+  class StoragePowerActor {
+   public:
+    static const power::Power kMinMinerSizeStor;
+    static const size_t kMinMinerSizeTarg;
+
+    static constexpr VMExitCode OUT_OF_BOUND{1};
+    static constexpr VMExitCode ALREADY_EXIST{2};
```

Comment:
To me it'd more natural, but it's up to you.
```suggestion
    static constexpr VMExitCode ALREADY_EXISTS{2};
```

Reply:
Fixed